week-1-ts-ecosystem/w01e02-enter-the-world-of-typescript/src/shop/cart-item-sum.component.ts

```diff
@@ -1,9 +1,9 @@
 import { div } from '../framework/dom-creators';
-import type { Price } from '../types/item';
+import { Price } from '../models/item';
 
-export function cartItemSum({ value, currency = 'PLN' }: Price): HTMLElement {
+export function cartItemSum({ value, currency = 'PLN' }: Price): HTMLDivElement {
   const $panelBlock = div('panel-block is-justify-content-end');
   $panelBlock.innerHTML = `Total price: ${value} ${currency}`;
 
-  return $panelBlock;
+  return $panelBlock as HTMLDivElement;
```

Comment:
ta asercja typów nie powinna tutaj być potrzebna `as` jeśli TS na Tobie @kdrzymalski ją wymusza, to oznacza że coś można ulepszyć w `dom-creators.ts`